rtengine/PF_correct_RT.cc

```diff
@@ -1059,8 +1059,8 @@ void ImProcFunctions::BadpixelsLab(LabImage * src, double radius, int thresh, fl
 
     // end luma badpixels
 
-    float ** tmaa = tmL; // reuse tmL buffer
-    const JaggedArray<float> tmbb(width, height);
+    float** const tmaa = tmL; // reuse tmL buffer
+    JaggedArray<float> tmbb(width, height);
```

Comment:
@Floessie Why did you remove the const here?

Reply:
@heckflosse Absolutely. But not tonight. I'll fix `JaggedArray<>` during this week (in which branch?), the compiler will tell us the places then...

Reply:
@Floessie Should those fixes to JaggedArray go into 5.4? If not, just change in my branch

Reply:
@heckflosse No, not for 5.4 as it's not a functional bug. I'll push it to this branch then.